wasabi/tests/test_jupyter.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+import subprocess
+import os
+import sys
+
+import wasabi
+
+TEST_DATA = Path(__file__).absolute().parent / "test-data"
+WASABI_DIR = Path(wasabi.__file__).absolute().parent.parent
+
+
+def test_jupyter():
+    # Make sure that the notebook can 'import wasabi', and will get the version of
```

Comment:
Even knowing the history it's quite hard to work out what this test is doing and what it's for. On first analysis you think the notebook is being converted but then not run. It would be helpful to add a bit more context/history in this comment.

Reply:
Good point -- I rewrote the comment to hopefully be clearer?